src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/DischargeToHospitalService.kt

```diff
@@ -48,7 +49,9 @@ class DischargeToHospitalService(
     return if (!lastMovement.isRelease()) {
       dischargeToHospital(offenderNo, dischargeTime, toLocation)
     } else {
+      // this is to support the "Migrate into Restricted Patients" process where an offender was released in NOMIS instead of being moved to hospital in Restricted Patients
       lastMovement.changeToHospitalDischarge(toLocation)
+      caseNoteMovementService.createReleaseNote(offenderBooking, lastMovement)
```

Comment:
Create another case note when we are migrating a released prisoner into Restricted Patients - so that it is consistent with other discharge to hospital use cases (and nicer for users because this case note includes the hospital whereas the release case note might not)